lib/Firtool/Firtool.cpp

```diff
@@ -90,7 +90,8 @@ LogicalResult firtool::populateCHIRRTLToLowFIRRTL(mlir::PassManager &pm,
       firrtl::createLowerMatches());
 
   // Width inference creates canonicalization opportunities.
-  pm.nest<firrtl::CircuitOp>().addPass(firrtl::createInferWidths());
+  //pm.nest<firrtl::CircuitOp>().addPass(firrtl::createInferWidths());
```

Comment:
Please either add a cmdline control over this or remove the old line.  Don't leave commented out code in the file.